test/PoolManager.t.sol

```diff
@@ -485,6 +489,117 @@ contract PoolManagerTest is TestSetup {
         centrifugeChain.updateTrancheTokenPrice(poolId, trancheId, currencyId, price, uint64(block.timestamp - 1));
     }
 
+    function testRemoveLiquidityPool()public {
```

Comment:
```suggestion
    function testRemoveLiquidityPool() public {
```

Reply:
surprised `forge fmt` didn't catch this one